### ISSUE TITLE
Basemap with fonts

### ISSUE THREAD

**Issue by wadclapp** [Leaflet/Leaflet.VectorGrid#187]
Hi

Are there any plans to add the drawing of fonts to the basemaps? So place names etc get displayed on the map?

Unless I'm missing something, the only solution I've found currently to achieve this with Vector Tiles in Leaflet is by using `mapbox-gl-leaflet` (which appears to be losing support) with `mapbox-gl` (which isn't ideal).

**Comment by wadclapp** [comment]
Hi @perliedman, thanks for the reply and info! Unfortunately, I wouldn't have the expertise to help in this area.

I would imagine Vector Tiles are soon becoming the preferred standard for tiling? It would be great if some sort of clean solution could be reached for it in Leaflet.

**Comment by tomchadwin** [comment]
Yeah, it's https://github.com/Leaflet/Leaflet.VectorGrid/pull/131, based on https://github.com/Leaflet/Leaflet.VectorGrid/pull/64.
